[2025-02-23T17:00:01.641+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T17:00:01.670+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: newyork_weather_pipeline.extract_weather_data scheduled__2025-02-23T16:00:00+00:00 [queued]>
[2025-02-23T17:00:01.682+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: newyork_weather_pipeline.extract_weather_data scheduled__2025-02-23T16:00:00+00:00 [queued]>
[2025-02-23T17:00:01.682+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-23T17:00:01.706+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): extract_weather_data> on 2025-02-23 16:00:00+00:00
[2025-02-23T17:00:01.714+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=21838) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-23T17:00:01.716+0000] {standard_task_runner.py:72} INFO - Started process 21848 to run task
[2025-02-23T17:00:01.717+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'newyork_weather_pipeline', 'extract_weather_data', 'scheduled__2025-02-23T16:00:00+00:00', '--job-id', '162', '--raw', '--subdir', 'DAGS_FOLDER/newyork_weather_pipeline/newyork_weather_etl.py', '--cfg-path', '/tmp/tmpc35ada80']
[2025-02-23T17:00:01.719+0000] {standard_task_runner.py:105} INFO - Job 162: Subtask extract_weather_data
[2025-02-23T17:00:01.798+0000] {task_command.py:467} INFO - Running <TaskInstance: newyork_weather_pipeline.extract_weather_data scheduled__2025-02-23T16:00:00+00:00 [running]> on host ee2081ae3a04
[2025-02-23T17:00:01.933+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='newyork_weather_pipeline' AIRFLOW_CTX_TASK_ID='extract_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-23T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-23T16:00:00+00:00'
[2025-02-23T17:00:01.934+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T17:00:01.957+0000] {base.py:84} INFO - Retrieving connection 'open_meteo_id'
[2025-02-23T17:00:02.597+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': 40.875, 'longitude': 74.125, 'generationtime_ms': 0.04112720489501953, 'utc_offset_seconds': 0, 'timezone': 'GMT', 'timezone_abbreviation': 'GMT', 'elevation': 3825.0, 'current_weather_units': {'time': 'iso8601', 'interval': 'seconds', 'temperature': '°C', 'windspeed': 'km/h', 'winddirection': '°', 'is_day': '', 'weathercode': 'wmo code'}, 'current_weather': {'time': '2025-02-23T17:00', 'interval': 900, 'temperature': -9.6, 'windspeed': 9.4, 'winddirection': 113, 'is_day': 0, 'weathercode': 1}}
[2025-02-23T17:00:02.641+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T17:00:02.642+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=newyork_weather_pipeline, task_id=extract_weather_data, run_id=scheduled__2025-02-23T16:00:00+00:00, execution_date=20250223T160000, start_date=20250223T170001, end_date=20250223T170002
[2025-02-23T17:00:02.735+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T17:00:02.757+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
